extensions/grpc-common/runtime/src/main/java/io/quarkus/grpc/common/runtime/Generated.java

```diff
@@ -0,0 +1,54 @@
+package io.quarkus.grpc.common.runtime;
```

Comment:
Minor but I think this annotation should be somehow considered API and so be in the `io.quarkus.grpc.common` package.

Reply:
Sure! 

I was wondering if we wanted to have a global "quarkus" `@Generated` annotation. Typically, Panache classes could use it (I'm not sure it will have any benefits).

Reply:
Fixed!